src/type-generator.ts

```diff
@@ -535,7 +535,8 @@ export class TypeGenerator {
 
   private typeForArray(propertyFqn: string, def: JSONSchema4): EmittedType {
     if (!def.items || typeof(def.items) !== 'object') {
-      throw new Error(`unsupported array type ${def.items}`);
+      // Falling back to an array of any type
```

Comment:
Do you have an example where `def.items` is defined but its type is not `object`? I wonder if we can differentiate this case and do something better. Like, can this exist?

```json
{
  "ArrayProp": { "type": "array", "items": "string" },
}
```

Reply:
Hey, I could find only [one instance](https://json-schema.org/understanding-json-schema/reference/array#additionalitems) where `items` is not `object`: 

```
{
  "type": "array",
  "prefixItems": [
    { "type": "number" },
    { "type": "string" },
    { "enum": ["Street", "Avenue", "Boulevard"] },
    { "enum": ["NW", "NE", "SW", "SE"] }
  ],
  "items": false
}
```

This is not working in Json2Jsii right now.

Reply:
Interesting. We'll skip handling that for now.